website/source/docs/post-processors/manifest.html.md

```diff
@@ -41,3 +41,26 @@ You can simply add `{"type":"manifest"}` to your post-processor section. Below i
   ]
 }
 ```
+
```

Comment:
I think showing what json generated this might be useful, as well as a more complex manifest, as with many post-processors etc

Reply:
the first is easy... the second will be helpful too but I'll have to actually run such a build so might take longer to create. Let's iterate.